Nodejs/Product/TestAdapterNetStandard/NetCoreTestDiscoverer.cs

```diff
@@ -70,73 +72,43 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
 
                 foreach (var (projectFile, propertyGroup) in projects)
                 {
-                    var testFramework = propertyGroup.Descendants(NodeProjectProperty.TestFramework).FirstOrDefault()?.Value;
```

Comment:
I didn't carefully read the old implementation, so I don't know if it did something different.  Let me know if I should.

Reply:
The only difference is that it considers now configuration files and has been simplified.